ui/src/containers/SampleGridTableContainer.jsx

```diff
@@ -758,15 +766,92 @@ class SampleGridTableContainer extends React.Component {
         }
         return null;
       });
-
     return sampleItemList;
   }
 
+  getPuckSampleItems(puck, sampleNumber) {
+    const sample = Object.values(this.props.sampleList).find(
+      (sample) => sample.location === puck + ':' + sampleNumber,
+    );
+    if (typeof sample === 'undefined') {
+      return null;
+    }
+    // const key = sample.sampleID;
+    const key = puck + ':' + sampleNumber;
+    const picked = this.props.inQueue(sample.sampleID) && sample.checked;
+
+    const classes = classNames('samples-grid-table-li', {
+      'samples-grid-table-item-selected': this.props.selected[key],
+      'samples-grid-table-item-to-be-collected': picked,
+      'samples-grid-table-item-collected': isCollected(sample),
+    });
+
+    const settings = {
+      dots: false,
+      infinite: false,
+      speed: 100,
+      slidesToShow: 6,
+      slidesToScroll: 6,
+    };
```

Comment:
This looks like a static object, so you can move it to the top of the file (and use constant case: `const SETTINGS = ...`) to avoid recreating it every time.